idea/task-web-client/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { useState, useEffect } from 'react';
+import Sessions from "./Sessions";
+import './App.css';
+
+function App() {
+    const [ sessions, setSessions ] = useState();
+    async function initSessions() {
+        const data = await fetch("https://localhost:5001/LoginMonitor", {
```

Comment:
Can we have url as a configuraion value to allow running on non local environment